### PR TITLE
Don't use deprecated getWorkbench method.

### PR DESCRIPTION
**Pull request by akurtakov**
Signed-off-by: Alexander Kurtakov <akurtako@redhat.com>

### CODE REVIEW

File: org.eclipse.wildwebdeveloper.feature/feature.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <feature
       id="org.eclipse.wildwebdeveloper.feature"
       label="%name"
-      version="0.7.2.qualifier"
+      version="0.7.3.qualifier"
```
Comment (PyvesB) [inline org.eclipse.wildwebdeveloper.feature/feature.xml]: Given that version 0.8.0 was just officially released, why are there older version numbers here?

Comment (akurtakov) [reply org.eclipse.wildwebdeveloper.feature/feature.xml]: @mickaelistria would have to say that :)

Comment (akurtakov) [reply org.eclipse.wildwebdeveloper.feature/feature.xml]: I'm going to merge this PR as it's semver related. E.g. version 0.8 is due to the API/extension point added in the xml bundle. I still think that feature version should reflect that though.

**Comment by mickaelistria** [comment]
Right, the feature version should be 0.8.0 for the release, and 0.8.1 now. I just forgot it.

**Pull request merged**